modules/swagger-codegen/src/main/resources/Java/libraries/feign/build.gradle.mustache

```diff
@@ -96,7 +96,7 @@ if(hasProperty('target') && target == 'android') {
 ext {
     swagger_annotations_version = "1.5.8"
     jackson_version = "2.7.5"
-    feign_version = "8.16.0"
+    feign_version = "8.17.0"
     junit_version = "4.12"
     oltu_version = "1.0.1"
```

Comment:
For `oltu version`, can we upgrade to 1.0.2?

https://mvnrepository.com/artifact/org.apache.oltu.oauth2/org.apache.oltu.oauth2.client